pkg/kubelet/nodestatus/setters_test.go

```diff
@@ -926,24 +927,12 @@ func TestReadyCondition(t *testing.T) {
 			},
 			expectConditions: []v1.NodeCondition{*makeReadyCondition(true, "kubelet is posting ready status. WARNING: foo", now, now)},
 		},
-		{
-			desc:             "new, not ready: runtime errors",
-			node:             withCapacity.DeepCopy(),
-			runtimeErrors:    []string{"foo", "bar"},
-			expectConditions: []v1.NodeCondition{*makeReadyCondition(false, "foo,bar", now, now)},
-		},
-		{
-			desc:             "new, not ready: network errors",
-			node:             withCapacity.DeepCopy(),
-			networkErrors:    []string{"foo", "bar"},
-			expectConditions: []v1.NodeCondition{*makeReadyCondition(false, "foo,bar", now, now)},
-		},
```

Comment:
Because of the return type of function signature changed from `string slice` to `error`. There is no need to test string slice case.

Moreover, If we want to compound many various errors of the network or runtime, we must write test code invoking `errors.Aggregate` such as:   
```golang
	node:             withCapacity.DeepCopy(),
	networkErrors:    utilerrors.NewAggregate([]error{errors.New("foo"), errors.New("bar")}),
	expectConditions: []v1.NodeCondition{*makeReadyCondition(false, "[foo,bar]", now, now)},
```

I thinks this will be tedious in test case, If you have some better solutions, I'll change this code.